app/api/tickets.py

```diff
@@ -45,7 +45,7 @@ def before_create_object(self, data, view_kwargs):
             event = self.session.query(Event).filter_by(id=view_kwargs['id']).one()
             data['event_id'] = event.id
 
-    decorators = (jwt_required, )
+    decorators = (is_registrar, )
```

Comment:
`is_registrar` is only for checking in attendees, use jwt_required for now. See into similar cases